tcf_core/settings/testing.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=import-error,unused-wildcard-import,wildcard-import
+"""Django settings module for testing environment such as Heroku"""
```

Comment:
IMO the name of this file should be something like `dev`, because at least to me "testing" connotes something along the lines of written test cases or a scratch file where some various settings were experimented with, which this is not.

Reply:
I still think it should be `testing` because we use Heroku to test major functionalities (possibly with the help of D&M) before deploying to production.
https://en.m.wikipedia.org/wiki/Deployment_environment#Environments.